elixir/apps/domain/lib/domain/config/definitions.ex

```diff
@@ -432,6 +433,13 @@ defmodule Domain.Config.Definitions do
     changeset: {Domain.Config.Configuration.ClientsUpstreamDNS, :changeset, []}
   )
 
+  @doc """
+  Delay time in milliseconds to wait before redirecting client on the sign in success page.
+
+  This is needed for acceptance tests.  In dev/staging/prod the default should work fine.
+  """
+  defconfig(:client_redirect_delay, :integer, default: 1)
```

Comment:
We don't need to expose it in definitions.ex unless this is something we want to control via an environment variable. It would be enough just to set it explicitly in `config.ex`.

Reply:
Ah, ok, I was thinking all config options had to be defined.  Good to know.  I'll remove this.